examples/CMakeLists.txt

```diff
@@ -141,6 +141,7 @@ add_sdl_example_executable(audio-simple-playback SOURCES audio/01-simple-playbac
 add_sdl_example_executable(audio-simple-playback-callback SOURCES audio/02-simple-playback-callback/simple-playback-callback.c)
 add_sdl_example_executable(audio-load-wav SOURCES audio/03-load-wav/load-wav.c DATAFILES ${CMAKE_CURRENT_SOURCE_DIR}/../test/sample.wav)
 add_sdl_example_executable(audio-multiple-streams SOURCES audio/04-multiple-streams/multiple-streams.c DATAFILES ${CMAKE_CURRENT_SOURCE_DIR}/../test/sample.wav ${CMAKE_CURRENT_SOURCE_DIR}/../test/sword.wav)
+add_sdl_example_executable(audio-planar-data SOURCES audio/05-planar-data/planar-data.c)
```

Comment:
Somehow, icculus' planar audio example got into this branch.
There are also other unwanted example/test changes.

Reply:
This is a good example of why I wanted this PR merged sooner; these mistakes happen in long-lived PRs.   :( 

Let's see about fixing up the merge/rebase to main and try and get last-minute concerns resolved.

Reply:
We rebased this pr on top of master, and this is no longer an issue :)